Source/WebService/Blackjack.WebService/ClientApp/src/app/components/play-game/play-game.component.html

```diff
@@ -0,0 +1,31 @@
+<div class="mainBody">
+  <div class="fundRelated">
+    <label>Your Balance: {{user.balance}}</label>
+    <label>Current wager: {{wager}}</label>
+    <div [class.hidden]="isPregameHidden" class="wagerBtns">
```

Comment:
I don't see a css class called wagerBtns in your scss file. Maybe an old class you ended up getting rid of? If it doesn't exist in the css anymore clean it up here so you don't commit and superfluous code to the main branch